schema/data/anchored/v0/classes/C07.jsonld

```diff
@@ -8,9 +8,13 @@
   "schema:name": "CarbonPlus Grasslands Credit Class",
   "schema:description": "This Credit Class provide incentive and a structure to significantly increase the amount of hectares/acres under managed grazing worldwide by providing land stewards with the necessary incentives to make this important work possible. The CarbonPlus credit focuses on soil carbon sequestration in grassland ecosystems. The Plus indicates that co-benefits (or secondary ecological benefits) allow for a credit that accounts for more than just carbon.",
   "rfs:hasPrimaryImpact": {
-    "rfs:impactType": "INCREASED_CARBON_SEQUESTRATION_STORAGE",
+    "rfs:hasImpactType": "INCREASED_CARBON_SEQUESTRATION_STORAGE",
     "schema:name": "Increased Carbon Sequestration and Storage"
   },
+  "rfs:managedUnderProgram": {
+    "schema:url": "'https://registry-program-guide.regen.network",
```

Comment:
Ditto - it looks like all of these registry-program-guide urls have the extra `'`

Reply:
I wonder if changing our core `url` attribute (with `slot_uri: schema:url`) to have a Range of [`URI`](https://linkml.io/linkml-model/latest/docs/Uri/) or [`Uriorcurie`](https://linkml.io/linkml-model/latest/docs/Uriorcurie/) would include validation for this. Maybe not this specific typo but likely others?